src/lib/createAdapter.ts

```diff
@@ -42,11 +42,13 @@ export function testCondition(
 			return answers[cond.name] === cond.value;
 		} else if ("contains" in cond) {
 			return (
+				answers[cond.name] &&
 				(answers[cond.name] as AnswerValue[]).indexOf(cond.contains) >
-				-1
+					-1
 			);
 		} else if ("doesNotContain" in cond) {
 			return (
+				!answers[cond.name] ||
```

Comment:
Why is that necessary? It has been working the entire time without it, even with optional questions. (or did we never create a situation where something depends on an optional question?)

Reply:
The situation is the following: if you have a widget only, it will cause an exception as the answer for adminFeatures doesn't exist, but it needs to check if it should ask for TabReact. This happened in the unit tests. You can try to revert those lines and run the unit tests, you should get the exception.

Reply:
I see - I probably never ran into this situation without also having a condition for "is adapter".